blocks/v2-columns/v2-columns.css

```diff
@@ -0,0 +1,66 @@
+.v2-columns {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.v2-columns .v2-columns-image,
```

Comment:
Avoid unnecessary specificity, for example this can be changed to: `.v2-columns-image, .v2-columns-content`

Check all the document ;)